recipes/cosmic-ray/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "cosmic-ray" %}
```

Comment:
Yeah, we should respect the original name if we can (e.g. no conflict). Even if it is ugly. 😞